src/snapshotter.js

```diff
@@ -94,12 +101,25 @@ export default class SnapShotter {
     }
   }
 
+  async executeScript(script) {
+    try {
+      const scriptToExecute = require(path.resolve(script));
+      await scriptToExecute(this._driver);
+    } catch (error) {
```

Comment:
I think you're missing a test case for this catch

Reply:
added :)